### PR TITLE
Quickstart Updates for .NET SDK v3.0.1

### PR DESCRIPTION
**Pull request by adrianhall**

### CODE REVIEW

File: client/xamarin.forms/Droid/MainActivity.cs [hunk 1]
```diff
@@ -1,4 +1,6 @@
 ﻿using System;
+using System.IO;
+using ZUMOAPPNAME;
```
Comment (dhei) [inline client/xamarin.forms/Droid/MainActivity.cs]: Is this reference necessary?

File: client/xamarin.forms/ZUMOAPPNAME/Constants.cs [hunk 2]
```diff
@@ -6,6 +6,8 @@ public static class Constants
 	{
 		// Replace strings with your mobile services and gateway URLs.
```
Comment (dhei) [inline client/xamarin.forms/ZUMOAPPNAME/Constants.cs]: Drop the 'gateway URLS' in this comment

File: client/xamarin.forms/Droid/MainActivity.cs [hunk 3]
```diff
@@ -21,8 +23,25 @@ protected override void OnCreate (Bundle bundle)
...
+			// SQLitePCL.Batteries.Init();
+
+			// Initialize Azure Mobile Apps
 			Microsoft.WindowsAzure.MobileServices.CurrentPlatform.Init();
+			
+			// Initialize Xamarin Forms
+			global::Xamarin.Forms.Forms.Init (this, bundle);
+
+			// Set up the TodoItemManager so that the appropriate file is used
+			Constants.OfflineDbPath = Path.Combine(
```
Comment (dhei) [inline client/xamarin.forms/Droid/MainActivity.cs]: Nitpick: It's not best practice to name a variable as constant but use it as a static variable. If you need to update it's value later, then don't name it constant. I would keep "localstore.db" in Droid/MainActivity.cs and iOS/AppDelegate.cs.

Comment (adrianhall) [inline client/xamarin.forms/Droid/MainActivity.cs]: Now that we are not doing the path adjustments, we can move it back into the main app.  Will do that in next checkin.

**Pull request merged**